### PR TITLE
chore(common/resources): Part 3 of additional HISTORY.md cleanup

### PR DESCRIPTION
**Pull request by darcywong00**
Part 3 of cleaning up HISTORY.md following #3942 and #3926

* Fix scopes that were `lmlayer` and `web/utils`
* Cleanup `common` scopes to be more specific (if possible)

### CODE REVIEW

File: HISTORY.md [hunk 1]
```diff
@@ -787,9 +787,9 @@
 
 ## 14.0.68 alpha 2020-05-06
 
-* fix(common): enable build for win x64, use global VERSION.md and fix decxstr() bug (#3076)
+* fix(common/core/desktop): enable build for win x64, use global VERSION.md and fix decxstr() bug (#3076)
 * change(android): Add methods to go between LanguageResource and JSON (#3079)
-* feat(common): auto-update for package versioning (#3078)
+* feat(common/resources): auto-update for package versioning (#3078)
```
Comment (jahorton) [inline HISTORY.md:792]: This one's debatable, but I think the original scope is more accurate.  It's not a resource-targeting script - it actually has direct effects on our final build products.

File: HISTORY.md [hunk 2]
```diff
@@ -828,59 +828,59 @@
 * chore(common/resources): bump @actions/http-client from 1.0.3 to 1.0.8 in /resources/build/version (#3047)
 * fix(oem/fv/ios): firstvoices icon and version info (#3044)
 * feat(windows): Sentry integration fixes and polish (#3006)
-* feat(common/lmlayer): compile joinWordsAt property (#3032)
+* feat(developer/compilers): compile joinWordsAt property (#3032)
 
 ## 14.0.60 alpha 2020-04-29
 
-* feat(common): initial use of lerna (in-repo package links only) (#2997)
+* feat(common/resources): initial use of lerna (in-repo package links only) (#2997)
```
Comment (jahorton) [inline HISTORY.md:835]: This one reflects why I consider the previous `common/resources` change to be debatable.  This one _is_ closer to `common/resources`, as it does _not_ directly affect build products; the changes at this point were development-targetted only.  I don't mind leaving this one as-is.

File: HISTORY.md [hunk 3]
```diff
@@ -828,59 +828,59 @@
...
-* feat(common/lmlayer): allow for verbose word breaker specification (#3023)
+* feat(developer/compilers): allow for verbose word breaker specification (#3023)
 
 ## 14.0.55 alpha 2020-04-23
 
 * change(android): Convert LanguageListActivity to utility (#3018)
 * refactor(web/engine): successful web-core compilation (#2992)
-* change(common/lmlayer): always bundle searchTermToKey() with model (#2971)
-* change(common/lmlayer): remove NFD table (#3014)
+* change(common/models/developer/compilers): always bundle searchTermToKey() with model (#2971)
```
Comment (jahorton) [inline HISTORY.md:865]:
```suggestion
* change(common/models, developer/compilers): always bundle searchTermToKey() with model (#2971)
```

- [ ] Required for my approval.  (Or a similar variant; pretty sure this was a typo.)

**Review by jahorton — CHANGES_REQUESTED** [review]

File: HISTORY.md [hunk 4]
```diff
@@ -415,7 +415,7 @@
 * feat(ios/app): universal links for keyboard installation (#3466)
 * fix(ios/engine): Proper associating-installer deinitialization, cancellation tests (#3468)
 * refactor(ios): error definitions, i18n (#3470)
-* chore: cleanup keyboardprocessor.h.in (#3473)
+* chore(common/core/web): cleanup keyboardprocessor.h.in (#3473)
```
[the remark this answers is not in the document]
Comment (darcywong00) [reply HISTORY.md]: good catch

File: HISTORY.md (hunk 1 above)
Comment (darcywong00) [reply HISTORY.md]: reverted to (common)

**Review by jahorton — APPROVED** [review]

**Pull request merged**